### ISSUE TITLE
Uncaught TypeError: report.getFilePath is not a function

### ISSUE THREAD

**Issue by Gert-dev** [twifty/atom-phpunit-integrator#17]
This error seems to occur when I double-click a test class name in the tests tree. Double-clicking the tests  themselves, in the nodes underneath it, work fine, though.

```
At /home/user/.atom/packages/php-unit-integrator/lib/views/php-unit-report-view.js:328

TypeError: report.getFilePath is not a function
    at handleDoubleClick (/packages/php-unit-integrator/lib/views/php-unit-report-view.js:328:22)
    at Function.module.exports.Emitter.simpleDispatch (/usr/lib/atom/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/usr/lib/atom/node_modules/event-kit/lib/emitter.js:141:28)
    at EtchTreeNode.value (/packages/php-unit-integrator/node_modules/colletch/components/etch-component.js:142:16)
    at EtchTreeNode.onClick (/packages/php-unit-integrator/node_modules/colletch/components/etch-tree-node.js:127:23)
```

**Comment by twifty** [comment]
I can reproduce, I'll push a fix shortly.

**Comment by twifty** [comment]
Fixed in 0.0.14

**Comment by twifty** [comment]
Hmm, I thought this was fixed but js seems to be keeping a reference to the last item in the loop. This is causing the click handler to open the wrong file.

**Comment by twifty** [comment]
Fixed (for real this time) in 0.0.15
